### DISCUSSION TITLE
Error to plsying libretube : confrim your nota Bot!!!!

### DISCUSSION THREAD

**Discussion by doolooloo** [libre-tube/LibreTube#6436] [Q&A]
Hello, it's been about a month that I can't watch YouTube videos in the program, the reason is because of the Tide warning for not being a robot, there is no way to solve this problem, I installed the old versions and logged in through the site, but there is no way to I confirm that I am not a robot, please guide me. I have been using this program for several years, and in my country, YouTube is a filter, and without this program, I cannot access YouTube. Please help me.

**Comment by doblejote** [comment]
me too
